internal/logger/target/http/http.go

```diff
@@ -415,25 +409,36 @@ func (h *Target) startQueueProcessor(ctx context.Context, mainWorker bool) {
 			}
 		}
 
+		var err error
 		if !isDirQueue {
 			err = h.send(ctx, buf.Bytes(), count, h.payloadType, webhookCallTimeout)
 		} else {
 			_, err = h.store.PutMultiple(entries)
 		}
 
 		if err != nil {
+			if errors.Is(err, context.Canceled) {
+				return
+			}
+
 			h.config.LogOnceIf(
 				context.Background(),
-				fmt.Errorf("unable to send webhook log entry(s) to '%s' err '%w': %d", name, err, count),
+				fmt.Errorf("unable to send audit/log entry(s) to '%s' err '%w': %d", name, err, count),
 				name,
 			)
 
-			if errors.Is(err, context.Canceled) {
-				return
+			if retryIntvl <= 0 {
+				retryIntvl = 3 * time.Second
```

Comment:
This can be initialized outside the loop so we don't have to 'if' for every loop